.github/workflows/ci.yml

```diff
@@ -336,6 +336,49 @@ jobs:
           path: reports-quarkus-base-unit-tests.zip
           if-no-files-found: ignore
 
+  quarkus-test-windows:
+    name: Quarkus Test - Windows
+    needs: build
+    runs-on: windows-latest
+    env:
+      MAVEN_OPTS: -Xmx2048m
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
```

Comment:
This could lead to clash of caches. I wonder whether GH runners are smart enough to convert it to proper windows path?

Reply:
Not sure either. Still checking.

Reply:
There is a cache miss for the mvn artifacts due to the fact that the build is run on linux and the entry key is calculated based on the runner OS.